.travis.yml

```diff
@@ -7,20 +7,10 @@ rvm:
   - 2.6
   - 2.5
   - 2.4
-env:
```

Comment:
I'm not sure about this..it's definitely way cleaner and faster in Travis but we don't run the test for each dry-v version which can be good since we can keep track of compatibility issue with a specific one.
What do you think?

Reply:
Those version are patch version of each other. 
There is api change only between 0.1.3 and 1.0.0 and soon 2.0.0